google-common/src/test/scala/docs/scaladsl/GoogleCommonDoc.scala

```diff
@@ -21,8 +21,7 @@ import pekko.stream.scaladsl.Source
 
 import scala.annotation.nowarn
 
-@nowarn("msg=never used")
-@nowarn("msg=dead code")
+@nowarn("msg=never used|dead code")
```

Comment:
This was necessary because for some reason the `@nowarn` annotation behaved differently on Scala 2.12 where it would complain if you had multiple `@nowarn` statements, i.e.

```
[info] compiling 1 Scala source to /Users/mdedetrich/github/incubator-pekko-connectors/google-common/target/scala-2.12/test-classes ...
[error] /Users/mdedetrich/github/incubator-pekko-connectors/google-common/src/test/scala/docs/scaladsl/GoogleCommonDoc.scala:26:7: class nowarn may not appear multiple times on class GoogleCommonDoc
[error] class GoogleCommonDoc {
```
Thankfully `msg` syntax accepts regex so this could easily be solved with regex or clause.